web_frontend/config/urls.py

```diff
@@ -10,4 +10,5 @@
     url(r'^api-auth/', include('rest_framework.urls', namespace='rest_framework')),
     url(r'^api/', include('osmaxx.excerptexport.rest_api.urls', namespace='excerptexport_api')),
     url(r'^job_progress/', include('osmaxx.job_progress.urls', namespace='job_progress')),
+    url(r'^pages/', include('osmaxx.core.urls', namespace='pages')),
```

Comment:
I don't like the `pages` namespacing in `osmaxx.example.com/pages/about/` and in `osmaxx.example.com/pages/downloads/`. (In the user-visible URLs. I don't care about the `namespace='pages'`.) Then again, we have to distinguish between the static downloads page and the user's list of downloads and the latter already occupies `osmaxx.example.com/downloads/`.

Or could we combine those two pages? The one-stop<del> shop</del><ins> URL</ins> for OSMaxx downloads, whether static or not. (Obviously with more content when logged in.)